extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -116,6 +116,7 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 	public _blobContainers!: azureResource.BlobContainer[];
 	public _refreshNetworkShareLocation!: azureResource.BlobContainer[];
 	public _targetDatabaseNames!: string[];
+	public _serverDatabases!: string[];
```

Comment:
nit: the public fields shouldn't have a `_` prefix, as that is meant to distinguish private/protected members.  It looks like other publics are already not following this convention so can be cleaned up in a different PR later.